docs/src/whatsnew/latest.rst

```diff
@@ -250,6 +250,9 @@ This document explains the changes made to Iris for this release
 #. `@wjbenfold`_ made :func:`iris.tests.stock.simple_1d` respect the
    ``with_bounds`` argument. (:pull:`4658`)
 
+#. `@lbdreyer`_ replaced `nose`_ with `pytest`_ as Iris' test runner.
+   (:pull:`4734`)
+
```

Comment:
I've added it in this position because it would have been merged between PR 4658 and 4503